templates/nextjs/src/lib.ts

```diff
@@ -1,11 +1,16 @@
 import { BN, WalletLocked, WalletUnlocked } from 'fuels';
 
-export const NODE_URL = `http://127.0.0.1:${
-  process.env.NEXT_PUBLIC_FUEL_NODE_PORT || 4000
-}/graphql`;
+export const CURRENT_ENVIRONMENT = process.env.NODE_ENV === 'development' ? 'local' : 'testnet';
+
+export const NODE_URL =
```

Comment:
@petertonysmith94 thanks for the suggestions! I have added a new env variable called `NEXT_PUBLIC_DAPP_ENVIRONMENT` which can be set to either `local` or `testnet`. Changing this single variable will change everything in the dapp accordingly.

I am not in favour of specifying the entire node url and the faucet url in the env. Reasoning:

We need to know only the _port_ to start the local node, not the whole URL. We just pass in the port to the fuels.config.ts file. We would have to strip off the port from the URL people specify. While on the other hand, we would need the whole URL for a testnet. This can cause confusion. 

I would much rather have the `DAPP_ENVIRONMENT` variable the single entrypoint that controls the 'environment', and then our source code can control the different URLs accordingly internally by reacting to this single variable.

Reply:
> I would much rather have the DAPP_ENVIRONMENT variable the single entrypoint that controls the 'environment', and then our source code can control the different URLs accordingly internally by reacting to this single variable.

That'd be the best, but then we'd have to enforce the user to _always_ run the node+faucet on the same ports predetermined by us, which IMO we shouldn't do.